index.html

```diff
@@ -2,12 +2,51 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Gonzalo's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;700&display=swap" rel="stylesheet">    
     <link rel="stylesheet" href="Assets/css/style.css" />
 </head>
 <body>
-    <h1>Gonzalo's Portfolio</h1>    
+    <header>
+        <div class="container header-flex">
+            <h1>Gonzalo A. Medina</h1>
+            <nav>
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <li><a href="#"><i class="far fa-envelope"></i></a></li>  
+                </ul>
+            </nav>
+            <div class="movil-menu">
+                <a href="#"><i class="fas fa-bars"></i></a>
+            </div>
+        </div>        
+    </header>
+    <main>
+        <div class="main-presentation">
```

Comment:
- [ ] As the previous reviewer suggested, You can use `section tag` after the `main tag` for better HTML semantics.